Python/Python.sublime-syntax

```diff
@@ -21,6 +21,8 @@ file_extensions:
   - gyp
   - gypi
   - Snakefile
+  - snakefile
```

Comment:
> In my case, `Snakefile` always works. However, when I looked more closely on the configuration files for Make to update the Snakemake configuration, I realized that I had originally forgotten to include other cases while Make has them.

I looks like we prefer removing the lowercased `makefile` from the `Makefile` syntax rather than adding lowercase `snakemake` in this PR.

Reply:
@karel-brinda The lowercased `makefile` extension has been removed from the syntax. If you can remove `snakefile` from this PR, it's good to go.